.github/workflows/build.yml

```diff
@@ -0,0 +1,163 @@
+name: Build
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          # GCC/G++
+          - os: ubuntu-16.04
+            CC: gcc
+            version: "4.8"
+            type: Debug
+          - os: ubuntu-16.04
+            CC: gcc
+            version: "4.8"
+            type: Release
```

Comment:
Out of curiosity: we used to use RelWithDebugInfo, here you use Release. Any particular reason?

Reply:
The difference between RelWithDebInfo and Release is whether debug symbols are included or not. But we always seem to have `-g` in CXX flags, so it does not make any difference. I'll keep Release for readability.